src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerDataSource.java

```diff
@@ -1104,27 +1104,27 @@
      * @param connectRetryCount
      *        maximum number of attempts
      */
-    public void setConnectRetryCount(int connectRetryCount);
+    void setConnectRetryCount(int connectRetryCount);
```

Comment:
I'm not sure why these were all explicitly set to public when none of the other public method are. Changed them back to be consistent.